Automattic/newspack-newsletters
# Update readme screenshot

### All Submissions:


### Changes proposed in this Pull Request:

This PR replaces the image used in one of the plugin screenshots to replace it with on licensed CC0; it also adds that information to the readme doc.

### How to test the changes in this Pull Request:

1. Apply PR.
2. Confirm the image in screenshot-2 has been updated to one with beach grass.
3. Confirm the information added to the readme about that image is correct.

### Other information:
